pkg/util/ipvs/testing/fake_test.go

```diff
@@ -113,46 +113,68 @@ func TestRealServer(t *testing.T) {
 		Port:     uint16(80),
 		Protocol: string("TCP"),
 	}
+	rsInfos := []struct {
```

Comment:
Why not construct `rss` directly here? `rsInfos` seem redundant here.

Reply:
I did this for same reason,,, but I can't remember it now,,,